drivers/sensor/apds9960/Kconfig

```diff
@@ -0,0 +1,41 @@
+# Kconfig - APDS9960 sensor configuration options
+
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig APDS9960
+        bool
+        prompt "APDS9960 Sensor"
```

Comment:
You can collapse lines 10-11 to:
``bool "APDS9960 Sensor"``

You can do this for all the other configs too.